src/phpDocumentor/Parser/Middleware/EmittingMiddleware.php

```diff
@@ -26,6 +26,7 @@
      */
     public function execute($command, callable $next)
     {
+        var_dump($command->getFile()->path());
```

Comment:
`var_dump()` intentional?

Reply:
Saw that one to. Fixing it currently :smiley: